assets/packages/adminsidepanel/src/components/subcomponents/_questionsgroups.vue

```diff
@@ -309,20 +309,21 @@ export default {
     <div id="questionexplorer" class="ls-flex-column fill ls-ba menu-pane h-100 pt-2">
         <div class="ls-flex-row button-sub-bar mb-2">
           <div class="scoped-toolbuttons-right me-2">
+           <button
+                class="btn btn-sm btn-outline-secondary me-2"
+                @click="collapseAll"
+                :title="translate('collapseAll')"
+            >
+              <i class="ri-link-unlink" />
+            </button>
             <button
                 class="btn btn-sm btn-outline-secondary"
                 @click="toggleOrganizer"
                 :title="translate(allowOrganizer ? 'lockOrganizerTitle' : 'unlockOrganizerTitle')"
             >
               <i :class="allowOrganizer ? 'ri-lock-unlock-fill' : 'ri-lock-fill'" />
             </button>
```

Comment:
not needed just order is reversed visually since float right is used, should be done without float and just use flex parameters, dont have to fix it :)